internal/output/utils.go

```diff
@@ -75,13 +87,126 @@ func MakeTable(rows []Row) ([]string, []Row) {
 				v.Name = h
 			}
 			row[i] = v
+			sv := fmt.Sprint(v.Value)
+			if len(sv) > width[i] {
+				width[i] = len(sv)
+			}
+			if !alignSet {
+				align[i] = alignmentForType(v.Type)
+			}
 		}
+		alignSet = true
 		rows[ri] = row
 	}
 	// remove this. prefix from header cells
 	thisPfx := fmt.Sprintf("%s.", NameValue)
+	columns := make(table.Row, len(header))
 	for i, h := range header {
-		header[i] = strings.TrimPrefix(h, thisPfx)
+		columns[i] = table.Column{
+			Header: strings.TrimPrefix(h, thisPfx),
+			Align:  align[i] * width[i],
+		}
+	}
+	return columns, rows
+}
+
+func makeTableHeaderFromRow(row Row, maxWidth int) table.Row {
+	dw := defaultWidthForFlexibleColumns(row, maxWidth)
+	// the max width for a column is 1000, just an arbitrary decision --YT
+	if dw > 1000 {
+		dw = 1000
 	}
-	return header, rows
+	ww := 0
+	for _, c := range row {
+		ww += widthForType(c.Type, len(c.Name), dw)
+	}
+	hd := make(table.Row, len(row))
+	for i, c := range row {
+		hd[i] = table.Column{
+			Header: c.Name,
+			Align:  alignmentForType(c.Type) * widthForType(c.Type, len(c.Name), dw),
+		}
+	}
+	return hd
+}
+
+func alignmentForType(t int32) int {
```

Comment:
For this function may be we can use enum for easy reading code.

```
const (
	LeftAlign    int = -1
	RightAlign  int = 1
)
```

Reply:
Added comments at: https://github.com/hazelcast/hazelcast-commandline-client/pull/165/commits/04a0368bd33191940eb258a57ba49b5ea45d97c9 Since this is a multipler, not sure making them enums would be beneficial.